src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/TestConfigurations/configuration-generalize-invalid-othervalues.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "fhirPathRules": [
+    {
+      "path": "Condition.onset as Age",
```

Comment:
what if customer do not need 'as Age'

Reply:
Condition.onset could be different types and user should specify it in config.

Reply:
Also works if no 'as Age', here is just an example.